firefox_desktop/views/crash_usage.view.lkml

```diff
@@ -1,38 +1,27 @@
 view: crash_usage {
-  sql_table_name: mozdata.telemetry.clients_daily ;;
-
-  # We use a Looker Filter so we can apply it directly to both crash_usage and crashes_daily views.
-  # Date filter predicate pushdown doesn't seem to be applied through the join (i.e.
-  # when we filter on submission_date here, because we join on submission_date with
-  # the crashes_daily view, we would expect the predicate filter to apply there as well.
-  # But BQ doesn't realize it's a predicate filter through the join, and fails the query
-  # since we require one on these tables)
-  filter: date {
-    type: date
-    default_value: "28 days"
-    description: "Submission date (date that we received the associated crash or usage data from the client)."
-  }
-
-  filter: sample_id {
-    type: number
-    description: "Sample ID to filter on. Use this to speed up iterative analysis."
-  }
+  sql_table_name: `moz-fx-data-shared-prod`.telemetry_derived.clients_daily_joined_v1 ;;
```

Comment:
Isn't `mozdata.telemetry.clients_daily` a downstream view of this table? I haven't looked much at the view definitions but I imagine it changes a lot in between? I don't follow how this removes the join parts